### PR TITLE
Actionbutton alone aria pressed

### PR DESCRIPTION
**Pull request by snowystinger**
I think this is what it's supposed to be, action button alone, use aria-pressed
action button in group, use aria-checked

Closes https://jira.corp.adobe.com/browse/RSP-1775

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [Issue](https://github.com/adobe-private/react-spectrum-v3/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Comment by snowystinger** [comment]
I'm not sure I like this
Maybe better to add a prop to actionbutton `isInGroup` or something along those lines that defaults false, when that's the case, it uses aria-pressed
when set to true, it uses aria-checked instead

File: packages/@react-aria/button/src/useButton.ts [hunk 1]
```diff
@@ -93,18 +93,22 @@ export function useButton(props: AriaButtonProps, ref: RefObject<HTMLElement>):
...
-      'aria-describedby': ariaDescribedBy,  
+      'aria-describedby': ariaDescribedBy,
       'aria-haspopup': ariaHasPopup,
       'aria-expanded': ariaExpanded || (ariaHasPopup && isSelected),
       'aria-controls': ariaControls,
-      'aria-checked': isSelected,
+      // if an explicit aria-pressed false has been passed int, then use that but as undefined
+      // otherwise, decide for ourselves
+      'aria-pressed': props['aria-pressed'] === false ? undefined : ariaPressed,
+      'aria-checked': props['aria-pressed'] === false ? isSelected : undefined,
```
Comment (devongovett) [inline packages/@react-aria/button/src/useButton.ts:111]: I don't think this is quite right. `aria-pressed` should be `undefined` if it is not a toggle button, and false if it *is* a toggle button but not pressed. Currently you have undefined if it's false as well. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Toggle_buttons.

**Comment by devongovett** [comment]
> Maybe better to add a prop to actionbutton isInGroup or something

Could you use the `role` prop for this? If the `role` is set to `checkbox` or `radio` then use `aria-checked`, otherwise `aria-pressed`?

File: packages/@react-aria/button/src/useButton.ts [hunk 2]
```diff
@@ -93,18 +93,19 @@ export function useButton(props: AriaButtonProps, ref: RefObject<HTMLElement>):
   let {focusableProps} = useFocusable(props, ref);
   let handlers = mergeProps(pressProps, focusableProps);
   let interactions = mergeProps(contextProps, handlers);
+  let ariaSelected = props['role'] === 'checkbox' || props['role'] === 'radio' ? 'aria-checked' : 'aria-pressed';
```
Comment (devongovett) [inline packages/@react-aria/button/src/useButton.ts:96]: nit: use dot syntax for role

**Pull request merged**